dev/windows/setup.bat

```diff
@@ -1,9 +1,17 @@
 @ECHO ON
-pushd %TEMP% || goto :error
+:: Obtain abs path to the conda source root
+pushd "%~dp0"
+    set "TWO_DIRS_UP=..\..\"
+    set "CONDA_SRC_PATH="
+    pushd %TWO_DIRS_UP% || goto :error
+        set "CONDA_SRC_PATH=%CD%"
+    popd || goto :error
+popd || goto :error
```

Comment:
🤯

Reply:
Yeah I know, at least we can have visual indentation to see what's going on 😂